wordle-web/src/views/WordleView.vue

```diff
@@ -17,21 +56,102 @@ import { ref, reactive, onMounted, onUnmounted } from 'vue'
 import GameBoard from '../components/GameBoard.vue'
 import KeyBoard from '../components/KeyBoard.vue'
 import type { Letter } from '@/scripts/letter'
+import Axios from 'axios'
 
 const guess = ref('')
 const game = reactive(new WordleGame())
 
+const overlay = ref(true)
+let nameDialog = ref(false)
+let endScreen = ref(false)
+let displayName = ref(game.displayName)
```

Comment:
I think you can use const here instead of using let.

Reply:
You're probably right, I think I changed them when messing with something else and forgot to see if I could change them back.